python/EC2_SECURITY_GROUP_NOT_USED/EC2_SECURITY_GROUP_NOT_USED_test.py

```diff
@@ -0,0 +1,160 @@
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import MagicMock
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::EC2::SecurityGroup'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+
+class Boto3Mock():
+    @staticmethod
+    def client(client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        if client_name == 'sts':
+            return STS_CLIENT_MOCK
+        raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('EC2_SECURITY_GROUP_NOT_USED')
+
+class SampleTest(unittest.TestCase):
+
+    #Gherkin Scenario 1: Security group is not associated with any ENI/EC2
+    def test_security_group_not_associated(self):
+        invoking_event = '{"configurationItem":{"configuration":{"groupName":"sg1"}, "relationships":[{"resourceId":"vpc-8b0422e2","resourceName":null,"resourceType":"AWS::EC2::VPC","name":"Is contained in Vpc"}], "configurationItemStatus":"OK", "resourceType":"AWS::EC2::SecurityGroup", "configurationItemCaptureTime":"2019-04-28T07:49:40.797Z", "resourceId":"sg-011af0a7f107d6256"}, "messageType":"ConfigurationItemChangeNotification"}'
+        response = RULE.lambda_handler(build_lambda_configurationchange_event(invoking_event), '{}')
+        resp_expected = []
+        resp_expected.append(build_expected_response("NON_COMPLIANT", "sg-011af0a7f107d6256", DEFAULT_RESOURCE_TYPE, "The security group sg1 is not associated with any ENI/EC2"))
```

Comment:
No need to mention DEFAULT_RESOURCE_TYPE as it's default value here: def build_expected_response(compliance_type, compliance_resource_id, compliance_resource_type=DEFAULT_RESOURCE_TYPE, annotation=None)